modules/core/system/lang_controller.py

```diff
@@ -0,0 +1,21 @@
+from core.command_param_types import Options
+from core.decorators import instance, command
+from core.tyrbot import Tyrbot
+
+
+@instance()
+class LangController:
+
+    def inject(self, registry):
+        self.ts = registry.get_instance("translation_service")
+        self.bot: Tyrbot = registry.get_instance("bot")
+
+    @command(command="lang", params=[Options(["de_DE", "en_US"])], description="Changes the language of the bot",
+             access_level="moderator", sub_command="set")
+    def reload_translation(self, request, lang):
+        self.bot.send_private_message(request.sender.char_id, "<orange>I'll change my language now... <end>")
```

Comment:
this shouldn't respond to the private channel, it should just return the message:

```
self.ts.reload_translation(lang)
return "Language changed to <highlight>%s<end>." % lang
```